### DISCUSSION TITLE
对不同语言分别设置不同的翻译显示效果

### DISCUSSION THREAD

**Discussion by fate1ess** [immersive-translate/immersive-translate#1382] [Ideas]
### 新功能 | New Feature

1.对繁体中文网页直接翻译
2.对英语网页显示双语翻译

### 为什么您想要添加这些功能 | Why you want to add these features

感觉翻译中文网页直接翻译显示效果好一些，一般不需要双语。

### 提议 | Proposal

当然我也可以设置不翻译繁体中文，然后配合 Chrome 自带的翻译功能，将繁体网页直接转换成简体中文。

**Comment by keyboard3** [comment]
1.2.0 已支持
![image](https://github.com/immersive-translate/immersive-translate/assets/13177744/4c6bc32c-e762-400b-8ba7-d45f83cd19ce)
